util/reggen/uvm_reg_base.sv.tpl

```diff
@@ -488,6 +488,9 @@ ${_create_reg_field(dv_base_prefix, reg_width, reg_block_path, reg.shadowed, reg
       reg_field_name = reg_name
     else:
       reg_field_name = reg_name + "_" + field.name.lower()
+
+    if reg.async_name and not reg.hwext:
```

Comment:
@cindychip @weicaiyang do you mind having a look at this?
basically the async registers now have a wrapper level on top of `prim_subreg`, so i'm guessing this is what we need to do.

Reply:
This LGTM, thanks @tjaychen
If this path is wrong, the `csr_hw_reset` test will fail as it checks the hierarchy path.